src/execution_plan/ops/op_node_by_label_scan.c

```diff
@@ -162,6 +163,15 @@ static OpResult NodeByLabelScanReset(OpBase *ctx) {
 	return OP_OK;
 }
 
+static OpBase *NodeByLabelScanClone(const ExecutionPlan *plan, const OpBase *opBase) {
+	assert(opBase->type == OPType_NODE_BY_LABEL_SCAN || OpType_NODE_BY_LABEL_AND_ID_SCAN);
```

Comment:
I think this assert is unnecessary, how else would it be possible to call it with an operation which is not OPType_NODE_BY_LABEL_SCAN or OpType_NODE_BY_LABEL_AND_ID_SCAN ?